recipes/artery-font-format/all/conanfile.py

```diff
@@ -0,0 +1,22 @@
+from conans import ConanFile, CMake, tools
+
+
+class ArteryFontFormatConan(ConanFile):
+    name = "artery-font-format"
+    version = "1.0"
+    license = "MIT"
+    homepage = "https://github.com/Chlumsky/artery-font-format"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Artery Atlas Font format library"
+    topics = ("conan", "artery", "font", "atlas")
+    no_copy_source = True
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version],
+                  strip_root=True, destination=self.source_folder)
```

Comment:
We (currently) usually extract the sources to a subdirectory, named `source_subfolder`.